src/contact/parallel/bvh_contact_manager.cc

```diff
@@ -201,9 +208,12 @@ struct NarrowphaseFunc
     // Size of offset = n + 1
     //
     std::vector<NarrowphaseResult> resa_vec, resb_vec;
+    ArborX::Experimental::TraversalPolicy policy;
+    policy._sort_predicates = false;
     //
-    a_bvh.query(nimble_kokkos::kokkos_device_execution_space{}, d_contact_nodes,
-              details::ArborXCallback{d_contact_faces, d_contact_nodes, contact_manager->GetPenaltyForceParam(), resa_vec, resb_vec});
+    a_bvh.query(nimble_kokkos::kokkos_host_execution_space{}, view_b,
+                details::ArborXCallback{view_a, view_b, contact_manager->GetPenaltyForceParam(), resa_vec, resb_vec},
+                policy);
```

Comment:
Alternatively,
```suggestion
                ArborX::Experimental::TraversalPolicy().setPredicateSorting(false));
```

Reply:
Thank you. I committed the first version.